components/stepper/index.css

```diff
@@ -170,29 +170,28 @@
 		--mod-textfield-border-color-disabled: var(--spectrum-stepper-border-color-disabled);
 	}
 
-	&:hover:not(.is-invalid, .is-disabled, .is-focused) {
+	&:hover:not(.is-invalid, .is-disabled, .is-focused, .is-keyboardFocused) {
 		--mod-infield-button-border-color: var(--mod-stepper-buttons-border-color-hover, var(--spectrum-stepper-buttons-border-color-hover));
 	}
 
 	&:not(.is-disabled) {
-		.is-focused,
+		&.is-focused,
```

Comment:
Since number field has been tricky as of late, can we update them? 

I'm pretty sure both of these selector changes were [updated in this PR](https://github.com/adobe/spectrum-css/pull/3558/files#diff-175c51cdabbdd380c802a29c593f17a0017bce1d76e2b9cc5e05d64c848a62b0) (we deprioritized a couple of months ago, that's all).